### ISSUE TITLE
Add S500+ (3600HB0302) support ? 🙏 

### ISSUE THREAD

**Issue by clemalex824** [jm-73/Indego#126]
Hi, 

I bought the S500 + version, the commands in HA are working fine (start, pause, return).
I don't have the battery status (unavailable).

Here is the information returned by the API:

```
alms//state
{'state': 258, 'map_update_available': True, 'mowed': 94, 'mowmode': 1, 'xPos': 93, 'yPos': 59, 'runtime': {'total': {'operate': 3201, 'charge': 929}, 'session': {'operate': 1, 'charge': 0}}, 'mapsvgcache_ts': 1622658502028, 'svg_xPos': 496, 'svg_yPos': 784, 'config_change': False, 'mow_trig': False}


alms//setup
{'hasOwner': True, 'hasPin': True, 'hasMap': True, 'hasAutoCal': False, 'hasIntegrityCheckPassed': True}


alms//updates
{'available': False}


alms//predictive
{'enabled': False}


alms//operatingData
{'runtime': {'total': {'operate': 3206, 'charge': 929}, 'session': {'operate': 6, 'charge': 0}}, 'battery': {'voltage': 9.4, 'cycles': 0, 'discharge': -0.2, 'ambient_temp': 30, 'battery_temp': 30, 'percent': 94}, 'garden': {'id': 1, 'name': 1, 'signal_id': 1, 'size': 202, 'inner_bounds': 0, 'cuts': 0, 'runtime': 3206, 'charge': 929, 'bumps': 207, 'stops': 4, 'last_mow': 3, 'map_cell_size': 120}, 'hmiKeys': 504}


alms//network
{'mcc': 208, 'mnc': 10, 'rssi': -88, 'currMode': 's', 'configMode': 's', 'steeredRssi': -100, 'networkCount': 3, 'networks': [20810, 20801, 20820]}


alms//map
b'<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 2066 1333"><rect x="0" y="0" width="2066" height="1333" r="0" rx="0" ry="0" fill="#FAFAFA" stroke="#FAFAFA"></rect><path d=" M 0 [......] 1872,960 " style="fill:#61B081;fill-rule:evenodd" />\n</svg>'


alms//config
{'region': 0, 'language': 12, 'border_cut': 1, 'is_pin_set': True, 'wire_id': 1, 'bump_sensitivity': 0, 'alarm_mode': True, 'mowing_mode': 0}


alms//calendar
{'sel_cal': 0, 'cals': [{'cal': 3, 'days': [{'day': 0, 'slots': [{'En': True, 'StHr': 10, 'StMin': 10, 'EnHr': 16, 'EnMin': 10}, {'En': False}]}, {'day': 5, 'slots': [{'En': True, 'StHr': 10, 'StMin': 10, 'EnHr': 16, 'EnMin': 10}, {'En': False}]}]}]}


alms//automaticUpdate
{'allow_automatic_update': False}


alms//serial
{'alm_sn': '<SERIAL>', 'service_counter': 2277, 'needs_service': False, 'alm_mode': 'manual', 'bareToolnumber': '3600HB0302', 'alm_firmware_version': '25116.01251'}


alms
[{'alm_sn': '<SERIAL>'}]


alerts
[]


Indego mower. State(state=258, map_update_available=True, mowed=94, mowmode=1, error=None, xPos=93, yPos=59, charge=None, operate=None, runtime=Runtime(total=RuntimeDetail(operate=32, charge=9, cut=23), session=RuntimeDetail(operate=1, charge=0, cut=0)), mapsvgcache_ts=1622658502028, svg_xPos=496, svg_yPos=784, config_change=False, mow_trig=False), , last mowed: None, next mow: None, None, None, [], map filename: None, Noneupdate available: False, State Descr: Docked.
```

![image](https://user-images.githubusercontent.com/35152089/120535115-e95acf80-c3e2-11eb-8d4f-ac7652b45e2e.png)

![image](https://user-images.githubusercontent.com/35152089/120535144-ef50b080-c3e2-11eb-86d3-54d65e229996.png)

![image](https://user-images.githubusercontent.com/35152089/120535179-f677be80-c3e2-11eb-8f8c-903b02e1299d.png)

**Comment by jm-73** [comment]
It is scheduled in the next release!

**Comment by jm-73** [comment]
Next release is up for code review. Stay tuned!

**Comment by jm-73** [comment]
Should be fixed in 3.8.0
